apps/openmw/switch_startup.cpp

```diff
@@ -130,6 +130,33 @@ void Switch::startup()
     appletLockExit();
 }
 
+void Switch::getUsername() {

```

Comment:
`std::string Switch::getUsername()`

Reply:
Damn it. I keep doing that for some reason